.gitignore

```diff
@@ -1,7 +1,12 @@
 # Data generated in the notebook directory. All committed data should be in the data/ folder.
-nbs/*.json*
-nbs/*.geojson*
-nbs/*.csv*
+nbs/**/*.json*
```

Comment:
Just adds more rules since the `usa_hcris2018_facilitybedcounts_20200313_v1.ipynb` pull down some ZIP archives that have a lot of different data types inside.